connect/runtime/src/test/java/org/apache/kafka/connect/runtime/distributed/IncrementalCooperativeAssignorTest.java

```diff
@@ -1508,25 +1545,25 @@ private List<Integer> allocations(Function<ConnectorsAndTasks, ? extends Collect
     private void assertNoRevocations() {
         returnedAssignments.newlyRevokedConnectors().forEach((worker, revocations) ->
                 assertEquals(
-                        "Expected no revocations to take place during this round, but connector revocations were issued for worker " + worker,
                         Collections.emptySet(),
-                        new HashSet<>(revocations)
+                        new HashSet<>(revocations),
```

Comment:
it can be `assertEquals(0, revocations.size)` or `assertTrue(revocations.isEmpty)`